jwst/exp_to_source/tests/helpers.py

```diff
@@ -1,17 +1,15 @@
 # Testing helpers
-from glob import glob
-import os
+from pathlib import Path
 
 
-INPUT_FILES_GLOB = 'data/jwst_nod?_cal.fits'
+INPUT_FILES_GLOB = "data/"
```

Comment:
with the change here, this is no longer really a "glob"; in fact, is this ever used anywhere now that the change is made?  If so, could you please point me to where?

Reply:
good catch! I removed the variable.